src/js/rbac/fetchPermissions.js

```diff
@@ -29,7 +29,7 @@ export const createFetchPermissionsWatcher = () => {
   let currentCall = {};
   return async (userToken, app = '') => {
     const user = await insights.chrome.auth.getUser();
-    if ([undefined, -1].includes(user.identity.account_number)) {
+    if ([undefined, -1].includes(user?.identity?.account_number)) {
```

Comment:
This would always pass, what do you think about this one
```suggestion
    if (user?.identity && [undefined, -1].includes(user.identity.account_number)) {
```

Reply:
Ah yeah I was not sure what is the actual condition. This makes sense